website/source/docs/builders/vmware-iso.html.markdown

```diff
@@ -108,6 +108,13 @@ builder.
     User's Guide](http://www.vmware.com/pdf/VirtualDiskManager.pdf) for desktop
     VMware clients. For ESXi, refer to the proper ESXi documentation.
 
+-   `disk_adapter_type` (string) - The adapter type of the VMware virtual disk
+    to create. The default is "lsilogic". This option is for advanced usage,
+    modify only if you know what you're doing. For more information, please
+    consult the [Virtual Disk Manager User's Guide]
+    (http://www.vmware.com/pdf/VirtualDiskManager.pdf) for desktop VMware
```

Comment:
I could not find anything equivalent to the PDF, but there are a couple of different places where the disk adapter type command line parameter is documented, depending on the hypervisor. For example,
https://www.vmware.com/support/developer/vcli/vcli41/doc/reference/vmkfstools.html
https://pubs.vmware.com/vsphere-50/index.jsp#com.vmware.vddk.utils.doc_50/diskutils_diskmgr.5.3.html
https://www.vmware.com/support/ws55/doc/ws_disk_manager_running.html

The PDF seems to mention all hypervisors, so maybe we can remove the text that says "refer to the proper ESXi documentation"?

The Virtual Disk Manager User's Guide was also linked in the `disk_type_id` attribute documentation (I basically just copied it), so maybe they can be consolidated in one place to minimize the duplicate text?